action_paymails.go

```diff
@@ -142,7 +142,7 @@ func (c *Client) DeletePaymailAddress(ctx context.Context, address string, opts
 	if err != nil {
 		return err
 	} else if paymailAddress == nil {
-		return ErrMissingPaymail
```

Comment:
What actually you trying to achieve with this change 😕

Reply:
Change the error message. Tried to align to the same message style in whole app.

Reply:
Why not just just change the string in ErrMissingPaymail? (or if this isn't readable enough, then rename ErrMissingPaymail). Because in your solution we have 2 errors which in code are used in exactly the same situations, so they mean exactly the same, but have different message.

Reply:
Changed message